src/common/socket.c

```diff
@@ -238,7 +238,11 @@ fd_set readfds;
 
 // Maximum packet size in bytes, which the client is able to handle.
 // Larger packets cause a buffer overflow and stack corruption.
-static size_t socket_max_client_packet = 24576;
+#if PACKETVER >= 20131223
+static size_t socket_max_client_packet = 0x10000;
```

Comment:
Thats something I would like to know. Did someone in the Herc team change it, or did someone in rA do it and then it got merged in here? I noticed that rA is applied the new size limit as well a day or 2 ago.